packages/modular-scripts/src/program.ts

```diff
@@ -245,13 +245,23 @@ program
     await port(relativePath);
   });
 
+const lintStagedFlag = '--staged';
 program
   .command('lint [regexes...]')
   .option(
     '--all',
     'Only lint diffed files from your remote origin default branch (e.g. main or master)',
   )
-  .option('--fix', 'Fix the lint errors wherever possible')
+  .option(
+    '--fix',
+    `Fix the lint errors wherever possible, restages changes if run with ${lintStagedFlag}`,
+  )
+  .addOption(
+    new Option(
+      lintStagedFlag,
```

Comment:
Tiny nit, the other options are just static strings, so for consistency's sake:

```suggestion
      '--staged',
```

Reply:
I saw that, however the `--fix` help text in line 257 of the same file also refers to `--staged`, and I thought in future if the `--staged` flag were to change, a developer might not realise that the `--fix` help text also needs updating. So I created this variable and used it in both places.

It's not a big deal as the usages of `lintStagedFlag` are very close together, so if you think it would be better to maintain consistency then that is fine with me.